package.json

```diff
@@ -56,26 +56,35 @@
   ],
   "dependencies": {
     "@fastify/busboy": "1.0.0",
+    "@sequelize/core": "7.0.0-alpha.47",
+    "@sequelize/sqlite3": "7.0.0-alpha.47",
```

Comment:
Another SQLite dependency? Maybe it's possible to replace `sqlite3` with this one.